tls_assets.go

```diff
@@ -4,10 +4,18 @@ package certificatetpr
 // These are used when getting a secret from the k8s API, to identify the component the secret belongs to
 type ClusterComponent string
 
+func (c ClusterComponent) String() string {
+	return fmt.Sprintf("%s", string(c))
```

Comment:
Why not just `return string(c)`? Formatting only with `%s` doesn't change the content at all.